jwst/pipeline/calwebb_tso3.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import unicode_literals, absolute_import
+
+import os
+
+from astropy.table import vstack
+
+from ..stpipe import Pipeline
+from .. import datamodels
+
+from ..outlier_detection import outlier_detection_tso_step
+from ..outlier_detection import outlier_detection_scaled_step
+from ..tso_photometry import tso_photometry_step
+from ..extract_1d import extract_1d_step
+from ..white_light import white_light_step
+
+__version__ = "0.7.8"
+
+
+class TSO3Pipeline(Pipeline):
```

Comment:
Extremely minor nit: Do we have a consensus on camel-case naming of classes, in terms of what we do with acronyms? Some folks, including myself, only capitalize the first letter of acronyms, e.g. Ami3Pipeline, just so that it's always only the first letter of a word that's capitalized. Is there a convention for this?